pkgs/applications/emulators/86box/default.nix

```diff
@@ -67,6 +80,7 @@ stdenv.mkDerivation rec {
   in
   ''
     wrapProgram $out/bin/86Box \
+      --unset QT_PLUGIN_PATH \
       "''${qtWrapperArgs[@]}" \
       --prefix ${libPathVar} : "${libPath}"
   '';
```

Comment:
I also think it doesn't make sense this `--unset`. Also, although it's not code you wrote, I must say I would have implemented this additional `--prefix` path completely different - I'd use `makeWrapperArgs+=(...)` in a `preFixup`, and I wouldn't use `dontWrapQtApps`.

Reply:
Done and working.